<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFundsDialog</class>
 <widget class="QDialog" name="AddFundsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddFundsDialog</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhFormattedNumbersOnly</set>
  </property>
  <widget class="QPushButton" name="addFundPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNameLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Card Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Funds:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fundValueLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhFormattedNumbersOnly</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currencyLineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Currency</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
